src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -290,7 +290,7 @@ struct Serializer
                         // Recurse into static array elements which contain a
                         // dynamic array.
 
-                        foreach (element; s.tupleof[i])
```

Comment:
This commit replaces
```d
foreach (i, T; typeof(s.tupleof)) { /* use s.tupleof[i] to access the i-th field of s */ }
```
with
```d
foreach (i, ref field; s.tupleof) { /* use `field` to access the i-th field of s */ }
```
So it changes both the `foreach` iteration statement to `ref field` and `s.tupleof[i]` to `field` in the loop body.
This particular line here adjusts `s.tupleof[i]` -> `field` that seems to have been missed when the same change had been applied in a previous commit.
Or did you spot a change in this commit that does something different?